### PR TITLE
10392 scaffold chapter36

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10392)

This is the PR for scaffolding chapter 36. This is only to scaffold, setting up the folder and file structure, and everything in this branch is subject to change.

## Testing done
Unit tests will be included in a separate branch

## Acceptance criteria
- [x] The initial pages have been created locally and a quick view has been shared with the team

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/config/form.js
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/form.js:78]: Not sure how this change got in here, but I think this snippet needs to be reverted.

File: src/applications/disability-benefits/686c-674/config/form.js
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/form.js:390]: These two snippets need to be reverted as well. Income and net worth questions need to remain out until post-launch.

**Review by micahchiang — APPROVED** [review]
Approved, the the changes to the 686 form config need to be reverted before merging.

**Review by rianfowler — APPROVED** [review]
You can update the codeowners file to avoid requiring a review from the FE review group for changes to this app.

File: src/applications/registry.json
Comment (micahchiang) [inline src/applications/registry.json:867]: Just realized, this replaces the current registry entry for chapter 31, which we still need. Instead, a new entry for chapter 36 should be added to the registry, probably just below this one.

**Review by micahchiang — CHANGES_REQUESTED** [review]

File: src/applications/registry.json
Comment (cohnjesse) [reply src/applications/registry.json:867]: the code for the existing chapter 31 is not in production nor will it ever be. What we are building is taking the place of that code.

**Review by micahchiang — APPROVED** [review]
LGTM. Thanks for making the change to registry.json!

**Pull request merged**
